.grabl/automation.yml

```diff
@@ -53,6 +53,7 @@ build:
       type: foreground
       command: |
         pyenv global 3.6.10
+        pip3 install -U pip
```

Comment:
Why are we upgrading `pip` in each run (as opposed to upgrading pip in our image)? (Or have we actually done both)

Reply:
@alexjpwalker I'll this to be a separate step in the image generation and then we can remove it